zulip/integrations/bridge_between_zulips/run-interrealm-bridge

```diff
@@ -9,6 +9,16 @@ import interrealm_bridge_config
 
 from typing import Any, Callable, Dict
 
+def check_subscription_or_die(zulip_client, stream):
+    # type: (zulip.Client, str) -> None
```

Comment:
The annotation needs to be changed to Python 3 version. Also applied to the other `check_subscription_or_die`.